docs/user-guide/SettingsFile.md

```diff
@@ -91,7 +91,11 @@ The path to the grammar.json file for the API in test.
 This is required when using the examples and payload body checkers.
 
 ### host: string (default None)
-Set to override the Host that's specified in the grammar
+Set to override the Host that's specified in the grammar.
+
+Ex: management.web.com
```

Comment:
>Ex [](http://example.com/codeflow?start=0&length=2)

Minor formatting: it might look better to quote it:

For example: ```management.web.com```

Reply:
Done